### DISCUSSION TITLE
(2일차) async await

### DISCUSSION THREAD

**Discussion by jjunhaa0211** [Info-iOS/iOStudy#144] [General]
async와 await 이제는 뭔지 보이기 시작하나요? 동시성을 공부할 때 봤을 것이라고 생각하는 이 문법 드디어 오늘 어떤 것인지 알아보는 좋을 시간이 되었으면 합니다. 그리고 보다 빠른 실력향상을 위해 남이 올린 글도 한번씩 읽으면 아무래도 잘하겠죠? 그리고 오늘 너무너무 바쁘다 그러면 다음날 아침 8시까지 제출 해주세요 😵

**Comment by Eunho0922** [comment]
넵
